lib/routes/bilibili/dynamic.ts

```diff
@@ -115,11 +116,16 @@ async function handler(ctx) {
     const useAvid = fallback(undefined, queryToBoolean(routeParams.useAvid), false);
     const directLink = fallback(undefined, queryToBoolean(routeParams.directLink), false);
 
+    const cookie = Object.values(config.bilibili.cookies).find(Boolean); // 寻找一个真值的 cookie，本路由中 cookie 可以是任意账号的
+    if (!cookie) {
+        throw new Error('缺少 Bilibili 用户登录后的 Cookie 值');
```

Comment:
Cookie is not compulsory.

Reply:
> Cookie is not compulsory.

At present, if cookies are not set, they will be subject to risk control. So cookies are necessary